From 38a48e7fd68973d2a1ccc1f7f59d5d0b59d2b74b Mon Sep 17 00:00:00 2001
From: Miklos Vajna <vmiklos@frugalware.org>
Date: Sat, 15 Mar 2008 12:27:31 +0100
Subject: [PATCH] Fix built with gcc-4.3 and newer

make[1]: Entering directory `/var/tmp/fst/src/dvd+rw-tools-7.1'
gcc -march=i686 -O2 -pipe -fno-strict-aliasing  -O2 -D_REENTRANT   -c -o growisofs.o growisofs.c
g++ -march=i686 -O2 -pipe -fno-strict-aliasing  -O2 -fno-exceptions -D_REENTRANT   -c -o growisofs_mmc.o growisofs_mmc.cpp
In file included from growisofs_mmc.cpp:17:
transport.hxx: In member function 'int Scsi_Command::is_reload_needed(int)':
transport.hxx:366: error: 'INT_MAX' was not declared in this scope
make[1]: *** [growisofs_mmc.o] Error 1
make[1]: Leaving directory `/var/tmp/fst/src/dvd+rw-tools-7.1'
make: *** [all] Error 2

This is becase limits.h is no longer automatically included in this
version.
---
 transport.hxx |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/transport.hxx b/transport.hxx
index 35a57a7..412aaaa 100644
--- a/transport.hxx
+++ b/transport.hxx
@@ -142,6 +142,7 @@ extern "C" char *plusminus_locale()
 #include <sys/ioctl.h>
 #include <linux/cdrom.h>
 #include <mntent.h>
+#include <limits.h>
 #include <sys/wait.h>
 #include <sys/utsname.h>
 #include <scsi/sg.h>
-- 
1.5.4.3

